Bindings~/java/src/test/java/com/alttester/TestsSampleScene1.java

```diff
@@ -61,1523 +63,1569 @@
 
 public class TestsSampleScene1 extends BaseTest {
 
-    @BeforeEach
```

Comment:
In this file it is more difficult to see what you've changed. Could you make sure you are using 4 spaces for indentation so that we don't have this problem in the future?